tests/end_to_end/netsh_test.cpp

```diff
@@ -201,16 +201,21 @@ TEST_CASE("show sections bpf.sys", "[netsh][sections]")
     std::string output = _run_netsh_command(handle_ebpf_show_sections, L"bpf.sys", nullptr, nullptr, &result);
     REQUIRE(result == NO_ERROR);
 
-    REQUIRE(
-        output == "\n"
-                  "                                    Size\n"
-                  "             Section       Type  (bytes)\n"
-                  "====================  =========  =======\n"
-                  "               .text       bind     1768\n"
-                  "\n"
-                  "                     Key  Value      Max\n"
-                  "          Map Type  Size   Size  Entries  Name\n"
-                  "==================  ====  =====  =======  ========\n");
+    const std::string expected_output = "\n"
+                                        "                                    Size\n"
+                                        "             Section       Type  (bytes)\n"
+                                        "====================  =========  =======\n"
+#if defined(NDEBUG)
```

Comment:
This `if` tests whether the unit tests were compiled for Debug or Release, not whether bpf.sys was compiled for Debug or Release.  As such, this test will fail in some manual cases.  It would be better to accept both sizes, not only one size.